numpy/linalg/linalg.py

```diff
@@ -1382,7 +1382,7 @@ def cond(x, p=None):
 
     Parameters
     ----------
-    x : (M, N) array_like
+    x : (..., M, N) array_like
```

Comment:
The SVD cond is defined even for non-square matrices, no? (frankly, did not try to put on my math brain for this)